src/components/templates/AuthLayout/AuthFormLayout.tsx

```diff
@@ -2,57 +2,103 @@ import { Alert, Box, Divider, Stack, Typography } from "@mui/material";
       </Typography>
-      <Typography variant="subtitle1" color="text.disabled" marginBottom={4.5}>
-        Your Social Campaigns
-      </Typography>
-      <Box display="flex" gap={1.25} marginBottom={4.25}>
-        <OutlineButton
-          startIcon={
-            <Box component="img" src="/icons/google-icon.svg" height="15px" />
-          }
+      {subtitle && (
+        <Typography
+          variant="subtitle1"
+          color="text.disabled"
+          marginBottom={4.5}
         >
-          Sign in with Google
-        </OutlineButton>
-        <OutlineButton startIcon={<IconApple />}>
-          Sign in with Apple
-        </OutlineButton>
-      </Box>
-      <Box width="100%">
-        <Divider>
-          <Typography
-            variant="subtitle2"
-            minWidth="max-content"
-            marginX={1.5}
-            color="text.disabled"
-            fontWeight="medium"
+          {subtitle}
+        </Typography>
+      )}
+      {!hideSocialButtons && (
+        <>
+          <Box
+            display="flex"
+            gap={1.25}
+            marginBottom={4.25}
+            sx={{ flexDirection: { xs: "column", md: "row" } }}
           >
-            Or with email
-          </Typography>
-        </Divider>
-      </Box>
+            <OutlineButton
+              startIcon={
+                <Box
+                  component="img"
+                  src="/icons/google-icon.svg"
+                  height="15px"
+                />
+              }
+            >
+              Sign in with Google
+            </OutlineButton>
+            <OutlineButton startIcon={<IconApple />}>
+              Sign in with Apple
+            </OutlineButton>
+          </Box>
+          <Box width="100%">
+            <Divider>
+              <Typography
+                variant="subtitle2"
+                minWidth="max-content"
+                marginX={1.5}
+                color="text.disabled"
+                fontWeight="medium"
+              >
+                Or with email
+              </Typography>
+            </Divider>
+          </Box>
+        </>
+      )}
       {error && (
-        <Stack sx={{ width: "100%", marginTop: 5 }} spacing={2}>
-          <Alert variant="outlined" severity="error" icon={false}>
-            {error}
+        <Stack sx={{ width: "100%", my: 3 }}>
+          <Alert variant="outlined" severity="error">
+            {Array.isArray(error)
+              ? error.map((err) => (
+                  <Typography
+                    variant="body2"
+                    color="text.secondary"
+                    key={`${err.message}$${err.field}`}
+                  >
+                    -{err.message}.
+                  </Typography>
+                ))
+              : error}
           </Alert>
         </Stack>
       )}
+      {successMessage && (
+        <Stack sx={{ width: "100%", my: 3 }}>
+          <Alert severity="success">{successMessage}</Alert>
+        </Stack>
+      )}
```

Comment:
Create or use lib for toasts passing queries through router seems a little extra